src/components/Sidebar/InternshipsSvg.tsx

```diff
@@ -0,0 +1,51 @@
+import { SvgIcon } from '@material-ui/core';
```

Comment:
I'd put this in `components/Icons` as it is re-usable.

Reply:
This is moved to appropriate folder